docs/home/whats-new-2-17.md

```diff
@@ -128,30 +128,69 @@ Deprecated features will be available for **two** versions ahead of the notifica
 
 The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. For more information about Run:ai's API support policy and deprecation process, see [Developer overview](../developer/overview-developer.md#overview-developer-documentation).
 
-The following list of API endpoints have been marked for deprecation:
+The following list of API endpoints and fields that have been marked for deprecation:
 
-#### Jobs, events, pods API (replaced by workloads/pods/events)
+#### Jobs and Pods API
 
-| Deprecated endpoint | Replacement endpoint |
+| Deprecated | Replacement |
 | -- | -- |
 | https://app.run.ai/v1/k8s/clusters/{uuid}/jobs | https://app.run.ai/api/v1/workloads |
```

Comment:
The prefix https://apps.run.ai is only relevant to SaaS. We dont know the prefix in self hosted so it is better to write https://[CP URL] or to just omit it

Reply:
I removed the prefix what so ever, and left it as it is in the API docs